components/fxa-client/android/src/main/java/mozilla/appservices/fxaclient/FirefoxAccount.kt

```diff
@@ -210,6 +210,15 @@ class FirefoxAccount(handle: FxaHandle, persistCallback: PersistCallback?) : Aut
         return getProfile(false)
     }
 
+    fun getEcosystemUserId(): String? {
```

Comment:
The name here should be `getEcosystemAnonId`; the `ecosystemUserId` field is different and we're not ready to expose that to consumers directly yet.